### DISCUSSION TITLE
Anyway to edit / remove the footer on the resume file ?

### DISCUSSION THREAD

**Discussion by LucasGenoud** [DeveloperPaul123/modern-cv#31] [Q&A] [answered]
As mentioned in the title, is there anyway to override or remove the footer in the resume ?
![image](https://github.com/DeveloperPaul123/modern-cv/assets/22954311/bda858b1-228e-4c14-b4df-ddd2922275b6)

The page number doesn't make a lot of sense since it's only one page anyway.

Best regards

**Comment by ptsouchlos** [comment]
Hello 👋 

This is a good point. Currently there is no way to do this but I should be able to add an option to facilitate this.

**Reply by LucasGenoud** [reply]
Hello,

I ended up forking the project and doing the modification locally as I had other changes I wanted to make to the template, like adding a profile picture in the resume template. I know they are highly discouraged in the US / UK but they are considered a good addition in my country. 

If I find the time I could look into properly integrating those changes in your project If you're interested ! 

Solid work on the template anyway this is by far the one I liked the most beside looking at numerous templates / CV maker webapp :)

**Reply by ptsouchlos** [reply]
> If I find the time I could look into properly integrating those changes in your project If you're interested !

That would be great! I'd like this template to be internationally useful, not just for the US/UK. If you're willing to file a PR that would be great. 

> Solid work on the template anyway this is by far the one I liked the most beside looking at numerous templates / CV maker webapp :)

Thank you! I really appreciate that 😁

**Comment by kir0ul** [comment]
Any news about this?
@LucasGenoud Could you provide the solution you used to remove the footer?

**Reply by ptsouchlos** [reply, marked as answer]
Implemented in #55
